src/github.ts

```diff
@@ -50,4 +52,35 @@ function EnhancedGitHubClient (options) {
   return octokit
 }
 
-module.exports = EnhancedGitHubClient
+export interface Options extends Octokit.Options {
+  debug: boolean
+  logger: Logger
+  limiter?: any
+}
+
+export interface OctokitRequestOptions {
+  method: string
+  url: string
+  headers: any
+}
```

Comment:
Can this and the next two interfaces be declared in the upstream octokit/rest.js repo?